mlflow/__init__.py

```diff
@@ -92,24 +90,25 @@
 
 _configure_mlflow_loggers(root_module_name=__name__)
 
-_major = 3
-_minor = 5
-_deprecated_version = (_major, _minor)
-_min_supported_version = (_major, _minor + 1)
-
-if sys.version_info[:2] == _deprecated_version:
-    warnings.warn(
-        "MLflow support for Python {dep_ver} is deprecated and will be dropped in "
-        "an upcoming release. At that point, existing Python {dep_ver} workflows "
-        "that use MLflow will continue to work without modification, but Python {dep_ver} "
-        "users will no longer get access to the latest MLflow features and bugfixes. "
-        "We recommend that you upgrade to Python {min_ver} or newer.".format(
-            dep_ver=".".join(map(str, _deprecated_version)),
-            min_ver=".".join(map(str, _min_supported_version)),
-        ),
-        FutureWarning,
-        stacklevel=2,
-    )
+# TODO: Uncomment this block when deprecating Python 3.6 support
```

Comment:
Note: python 3.6 will reach EOL on 2021/12/23:
https://devguide.python.org/#status-of-python-branches